repos/system_upgrade/common/actors/setetcreleasever/libraries/setetcreleasever.py

```diff
@@ -0,0 +1,22 @@
+from leapp.libraries.stdlib import api
+from leapp.libraries.common.config.version import get_source_major_version
+
+
+def _get_default_pkg_manager():
+    return "yum" if get_source_major_version() == '7' else "dnf"
+
+
+def _set_releasever(releasever):
+    releasever_path = '/etc/{}/vars/releasever'.format(_get_default_pkg_manager())
```

Comment:
no need to guess whether yum or dnf is used. The target OS has always DNF. Btw, it could be defined as constant. As well `yum` paths are symlinked to dnf.

```suggestion
def _set_releasever(releasever):
    releasever_path = '/etc/dnf/vars/releasever'
```